instrumentation/kotlinx-coroutines/kotlinx-coroutines-1.0/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+description: >
+  This instrumentation enables context propagation for Kotlin coroutines and adds support for
+  @WithSpan annotations on Kotlin suspend functions, it does not emit any telemetry on its own.
```

Comment:
I'm not sure about `it does not emit any telemetry on its own` since spans are created by `@WithSpan`.